pkgs/development/tools/language-servers/emmet-language-server/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildNpmPackage rec {
+  pname = "emmet-language-server";
+  version = "v2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "olrtg";
+    repo = pname;
```

```suggestion
    repo = "emmet-language-server";
```
Usually the change of `pname` shouldn't influence the change in source. More info https://github.com/nix-community/nixpkgs-lint/issues/21